LICENSE.txt

```diff
@@ -12,18 +12,18 @@ the parts that you might want to include with your own program.
  * The annotations and utility files are licensed under the MIT License.
    (The text of this license also appears below.)  This applies to the
    checker-qual*.jar and all the files that appear in it:  every file in a
-   qual/ directory, plus utility files including FormatUtil.java,
+   qual/ directory, plus utility files FormatUtil.java,
    I18nFormatUtil.java, NullnessUtil.java, Opt.java,
    PurityUnqualified.java, RegexUtil.java, SignednessUtil.java, and
    UnitsTools.java.  It also applies to other utility files, such as
    SignednessUtilExtra.java, and to the cleanroom implementations of
```

Comment:
Oops, I missed one of the "weasel words" earlier. ;) Other than this, everything looks great. Thanks!

```suggestion
   UnitsTools.java.  It also applies to other utility files:
   SignednessUtilExtra.java and the cleanroom implementations of
```